### PR TITLE
Qradar indicator hunting fix

### PR DESCRIPTION
**Pull request by ArikDay**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Qradar Indicator Hunting V2 playbook fixes:
1. The deprecated command 'qradar-get-assets' has been replaced with 'qradar-assets-list.'
2. Added the missing task 'Qradar Full Search on MD5' under the 'Qradar query without custom fields' section.3.
3.  Fixed the name of task 173.
4. Re-arranged the 'Set Detected Values' section to avoid errors.
5. Fixed several task inputs by removing unnecessary curly brackets.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by AdiPeret — APPROVED** [review]

File: Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml [hunk 1]
```diff
@@ -698,14 +699,10 @@ tasks:
...
       iscommand: false
-      brand: ''
+      brand: ""
     nexttasks:
-      '#default#':
-      - '164'
-      'yes':
-      - '2'
-      - '5'
-      - '6'
```
Comment (altmannyarden) [inline Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml:708]: missing else condition

File: Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml [hunk 2]
```diff
@@ -1585,7 +1533,7 @@ tasks:
             args:
               key:
                 value:
-                  simple: QRadar.Search.Result.events
+                  simple: QRadar.SearchEvents.Events
```
Comment (altmannyarden) [inline Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml:1536]: was the output of the previous command changed?

File: Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml [hunk 3]
```diff
@@ -1555,25 +1503,25 @@ tasks:
...
       description: Sets a value into the context with the given context key
       scriptName: SetAndHandleEmpty
       type: regular
       iscommand: false
-      brand: ''
+      brand: ""
     nexttasks:
       '#none#':
-      - '162'
+      - "163"
```
Comment (altmannyarden) [inline Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml:1521]: 162 is get hosrname and 163 is to set the hostnames. 

I think we should change the order (162 first).

File: Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml (hunk 2 above)
Comment (ArikDay) [reply Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml:1536]: Yes

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**